cmd/argo/commands/archive/list.go

```diff
@@ -25,6 +25,18 @@ func NewListCommand() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "list",
 		Short: "list workflows in the archive",
+		Example: `# List all archives:
+  argo archive list
+
+# List all archives in JSON format:
+  argo archive list -o json
+
+# List archives filtered that have both labels:
+  argo archive list -l key1=value1,key2=value2
+
+# List archives fetched in chunks of 100:
+  argo archive list --chunk-size 100
```

Comment:
The subject is not "archives" and there is only one "archive". The term you're looking for might be "archived workflows"

```suggestion
		Example: `# List all archived workflows:
  argo archive list

# List all archived workflows fetched in chunks of 100:
  argo archive list --chunk-size 100

# List all archived workflows in YAML format:
  argo archive list -o yaml

# List archived workflows that have both labels:
  argo archive list -l key1=value1,key2=value2
```

- Regarding the label examples, match the wording from #12706
  - use `-o YAML` same as #12706 as a more common format as well
- move the chunks to the 2nd example, as they may be quite useful